changelog/Dockerfile

```diff
@@ -1,4 +1,5 @@
 FROM tufin/oasdiff:main
+RUN apk add --no-cache expect
```

Comment:
It is less relevant for this PR, right?
Please, add it to all Dockerfile(s) or create a dedicated PR for that.

Reply:
The `expect` package contains the `unbuffer` executable